scripts/perf-activation-session.py

```
@@ -202,22 +236,125 @@ def stop_app(self) -> None:
+
+    def run_cli(
+        self,
+        args: list[str],
+        input_text: str | None = None,
+        timeout: float = 60,
+        check: bool = True,
+        socket_retries: int = 0,
+    ) -> str:
+        last_proc: subprocess.CompletedProcess[str] | None = None
+        for attempt in range(socket_retries + 1):
+            proc = subprocess.run(
```

Ruff (0.15.15)

[error] `subprocess` call: check for execution of untrusted input

(S603)